Changes

```diff
@@ -229,6 +229,11 @@ Working version
 - #13578: On Windows, use the OS CSPRNG to seed the Stdlib.Random generator.
   (Antonin Décimo, review by Miod Vallat, Nicolás Ojeda Bär, and Xavier Leroy)
 
+- #13843: Add signal definitions for SIGIO, SIGWINCH and SIGINFO. Introduces a
```

Comment:
I think strictly speaking this is a breaking change, so it should be prefixed with an asterisk.
```suggestion
* #13843: Add signal definitions for SIGIO, SIGWINCH. Introduces a
```